src/main/java/com/oneops/proxy/model/AppGroup.java

```diff
@@ -136,7 +136,7 @@ public String getNsPath() {
    */
   public String getGroupName() {
     if (null != org || null != assembly || null != env) {
-      return String.format("/%s/%s/%s/%s", domain.getType(), org, assembly, env);
+      return String.format("/%s/%s/%s/%s", domain.getType(), org, assembly, env).toLowerCase();
```

Comment:
Is this a backward compatible change? What happens existing apps created with upper case org/assembly/env ?

Reply:
Oneops allows creating env in upper-case. When secret-proxy manages secrets then Operations like Get, Details, Update and Delete fails because it invokes getGroupName() method. Using uppercase fails with this error :
vn0ytch$ ./secrets-1.1.0 get  -a oneops_secreproxy-varsha_QA-TEST myval.properties
✗ Secret myval.properties not found.